packages/AlgernonTests.package/ALGTestLifecycle.class/instance/testAlgernonChangeShortcut..st

```diff
@@ -1,6 +1,6 @@
 testing
 testAlgernonChangeShortcut: shortcut 
-	ALGCore activationString: shortcut shallowCopy.
+	ALGCore setKeyboardBinding: shortcut shallowCopy.
```

Comment:
Is here a reason for `shallowCopy` here. `Object>>copy` is defined as `self shallowCopy postCopy`.